sp800-63a/sec5_proofing.md

```diff
@@ -35,23 +37,24 @@ This section provides requirements on the properties and qualities of identity e
+|Strong|- The issuing source of the evidence confirmed the claimed identity through written procedures designed to enable it to form a reasonable belief that it knows the true identity of the subject. Such procedures shall be subject to recurring oversight by regulatory or publicly accountable institutions. For example, the Customer Identification Program guidelines established in response to the USA PATRIOT Act of 2001 or the [Red Flags Rule](#rfr), under Section 114 of the Fair and Accurate Credit Transaction Act of 2003 (FACT Act)<br><br>- The issuing process for the evidence ensured that it was delivered into the possession of the subject to whom it relates.<br><br>  -The issued evidence contains at least one reference number that uniquely identifies the subject to whom it relates.<br><br>- The applicant's name on the issued evidence must be the name that the identity was officially known at the time of issuance. Pseudonyms, aliases, and initials for last name and at least one first name are not permitted. **MG: I can't parse this. Can we split it into two requirements?Is initial not allowed for last name at all and allowed for no more than one first name? Also, should we refer to "last or family name" and "first or given name"?**<br><br>- The issued evidence contains a photograph, image, or biometric of the person to whom it relates.<br><br>- Where the issued evidence includes digital information, that information is protected using cryptographic or proprietary methods, or both, and those methods ensure the integrity of the information and enable the authenticity of the claimed issuing source to be confirmed.<br><br>  -Where the issued evidence contains physical security features, it requires proprietary knowledge and proprietary equipment to be able to reproduce it.<br><br>- The evidence is unexpired.|
```

Comment:
Recommend removing the MG comment. Probably need to be careful about use of, for example, last name vs. family name vs. surname -- consider Chinese family names that come first and Icelandic patronymic naming, for example.  I'm thinking it should be surname and given name. Also a bit of concern about some Indian individuals who use a last initial to avoid caste indication.